### PR TITLE
Add elasticsearch dump script

### PR DESCRIPTION
**Pull request by ylamgarchal**
This commit adds the elastic dump script which
dump the elasticsearch database in the local
file system. The dump will be uploaded in swift
in the same manner as the other logs.

### CODE REVIEW

File: virtualize.sh [hunk 1]
```diff
@@ -148,6 +148,12 @@ ssh $SSHOPTS root@$installserverip "
 
 kill ${tail_job}
 
+# Dump elasticsearch logs into /var/log/elasticdump,
+# upload_logs will update the dump in swift.
+scp $SSHOPTS $ORIG/dumpelastic.py root@$installserverip:/tmp
```
Comment (goneri) [inline virtualize.sh]: Do you think it's possible to adjust `dumpelastic.py` in order to be able to call it directly from the local machine? For example, by adding a `--target-server=$installserverip argument`.

Comment (goneri) [inline virtualize.sh]: I mean, Why don't we just run `./dumpelastic.py --url=http://$installserverip:9200` from the local machine?

Comment (ylamgarchal) [inline virtualize.sh]: Well, the thing is that ./dumpelastic.py pull the document from the database and then dispatch each log message in its corresponding file.

If we do that, we will pull the log from the installserver and then push them in the install server's FS which is not efficient at all :/.

File: dumpelastic.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import os
+import sys
+
+import requests
```
Comment (goneri) [inline dumpelastic.py:22]: You should add it in the `requirements.txt`.

**Comment by goneri** [comment]
Can you also add some unit-test. I don't think we need a 100% coverage, but at least a test to ensure the script still compile would be a good thing :)

**Pull request merged**